src/aes.rs

```diff
@@ -124,6 +124,23 @@ impl<R: Read> AesReader<R> {
             finalized: false,
         })
     }
+
+    /// Read the AES header bytes and returns the verification value and salt.
+    ///
+    /// # Returns
+    ///
+    /// the verification value and the salt
+    pub fn get_verification_value_and_salt(mut self) -> io::Result<(Vec<u8>, Vec<u8>)> {
```

Comment:
Change the verification value to `[u8; PWD_VERIFY_LENGTH]` since its size is constant.